assets/src/application/ui/display/withTooltip/style.scss

```diff
@@ -17,7 +17,7 @@
 	position: relative;
 	width: auto;
 	word-spacing: 0.15rem;
-	z-index: 1401;
+	z-index: 100000;
```

Comment:
I tried different values and only starting with this one it started to appear above the underlying modal.

Reply:
the proof:

![Peek 2020-06-24 16-45](https://user-images.githubusercontent.com/1477453/85568355-49c18880-b63a-11ea-8aca-5dae5a60e2d2.gif)